### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by donat**

### CODE REVIEW

**Comment by donat** [comment]
@bot-gradle test RFN

File: platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r87/ProblemProgressEventCrossVersionTest.groovy [hunk 1]
```diff
@@ -125,14 +125,15 @@ class ProblemProgressEventCrossVersionTest extends ToolingApiSpecification {
...
+                .forTasks(":reportProblem")
                 .setJavaHome(jdk17.javaHome)
                 .addProgressListener(listener)
-                .get()
+                .run()
         }
 
         then:
-        thrown(BuildException)
+        def  e = thrown(BuildException)
```
Comment (hegyibalint) [inline platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r87/ProblemProgressEventCrossVersionTest.groovy]:
```suggestion
        thrown(BuildException)
```
🤔 I think you might've used this but not anymore

**Comment by donat** [comment]
@bot-gradle test ReadyForRelease

File: platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r85/ProblemProgressEventCrossVersionTest.groovy [hunk 2]
```diff
@@ -53,7 +53,7 @@ class ProblemProgressEventCrossVersionTest extends ToolingApiSpecification {
 
         then:
         thrown(BuildException)
-        def problems = listener.problems.collect { new JsonSlurper().parseText(it.json) }
+        def problems = listener.problems.collect { new JsonSlurper().parseText(it.descriptor.json) }
```
Comment (hegyibalint) [inline platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r85/ProblemProgressEventCrossVersionTest.groovy:56]: 🤔 It's over me how this was working at all if this accessor was wrong...

**Review by hegyibalint — APPROVED** [review]

**Comment by reinsch82** [comment]
@bot-gradle test ACT

**Review by hegyibalint — APPROVED** [review]

**Review by donat — COMMENTED** [review]
Lgtm

**Pull request merged**